frontend/components/HeaderCom.vue

```diff
@@ -67,5 +67,9 @@ export default {
     position: sticky;
     z-index: 2;
     border-radius: 0;
+    background-color: #0079CF;
```

Comment:
Gebruik een config file zodat er een centrale plek is waar je al je kleuren kan beheren